### ISSUE TITLE
Allow Geo and time dimensions to be NULL at the configuration

### ISSUE THREAD

**Issue by zeginis** [Swirrl/graphql-qb#108]
In some dataset the geo and time dimensions are not defined OR they do not allign with the CubiQL requirements e.g. time dimension may not have `hasBegining `and `hasEnd`. In this case CubiQL should handle geo and time dimensions like every other dimension (i.e. enum dimensio).

We should extend the CubiQL configuration to define the "unmapped dimensions" geo and time as NULL.

**Comment by lkitching** [comment]
@zeginis  - This should now work if you set `:geo-dimension-uri` and `:time-dimension-uri` to `nil` in the configuration.

**Comment by zeginis** [comment]
Should we also chage these to return nil by default?

```
(defn geo-dimension [{config-geo :geo-dimension-uri :as config}]
  ;;Return the default sdmx:refArea if :geo-dimension is not defined at configuration
  (if (nil? config-geo)
    sdmx:refArea
    (URI. config-geo)))

(defn time-dimension [{config-time :time-dimension-uri :as config}]
  ;;Return the default sdmx:refPeriod if :time-dimension is not defined at configuration
  (if (nil? config-time)
    sdmx:refPeriod
    (URI. config-time)))
```

I tried with nil at the configuration for both dimensions but it does not work.

**Comment by lkitching** [comment]
@zeginis - Yes we should remove the defaults as well. Does it work if you do that?

**Comment by zeginis** [comment]
Yes by removing the defaults it work ok.

**Comment by lkitching** [comment]
Fixed by #130
